src/router.js

```diff
@@ -2,7 +2,9 @@ const { Router } = require("express");
 const controller = require("./controller");
 
 const router = Router();
-router.post("/api/shorturl", controller.postEntry);
-router.get("/api/get-url/:shorturl", controller.getEntry);
-router.get("/api/redirect/:shorturl", controller.getRedirect);
+router.post("/api/shorturls", controller.postEntry);
+router.get("/api/shorturls/:shorturl", controller.getEntry);
+// Does not consider this endpoint as 'headless' api, but a link that would be
```

Comment:
Is the change from "/api/redirect/:shorturl"  to  "/open/:shorturl" (with regards to the specified endpoint presenting a headless API functionality) a naming convention/stylistic OR am I missing something more essential?

Reply:
It's a naming convention. It can be /redirect/:shorturl also. Having it prefaced with 'API' could mislead a developer/user. if we plan to use that URL in the browser, it should be user-friendlier. If it's meant to be programmatically used, then it can be prefaced.